### PR TITLE
docs: update Overview docs

### LINKED ISSUE #82: Update Catalyst overview page on documentation website
Tasks 

- [x] @rymorgan write draft 
- [x] @machikoyasuda review draft 
- [x] @reaction-justin review draft 
- [x] @rymorgan final approval
- [ ] @machikoyasuda add to site 

Here's the draft in notion. 
https://www.notion.so/reactioncommerce/Draft-About-Catalyst-Design-System-8272c97d54e84f4d815fe0bb6d4705d8

### PR DESCRIPTION
**Pull request by machikoyasuda**
Signed-off-by: Machiko Yasuda <machiko@reactioncommerce.com>

Resolves #82 
Impact: **minor**  
Type: **docs**

## Docs
- Update Introduction/Copy copy

## Breaking changes
none

## Testing
1. Check it against the Notion doc

### CODE REVIEW

**Comment by machikoyasuda** [comment]
@rymorgan ready for review

File: styleguide/src/sections/Introduction.md [hunk 1]
```diff
@@ -1,13 +1,30 @@
-Introducing Reaction Commerce's **Catalyst Design System** and the `@reactioncommerce/catalyst` NPM module, built with React, React Hooks, Material-UI, JSS and the React Testing Library. The **Catalyst Design System** is a lightweight open source design system for building Reaction Admin tools and plugins.
+**Catalyst** is [Reaction Commerce](https://www.reactioncommerce.com/)'s lightweight design system that we use to build Reaction's Admin interface. Catalyst components are used for Reaction Admin and any future operator user interfaces we may build. The Catalyst components are focused on the user experience for operators and will help consistency across the Reaction Admin. Additionally, Catalyst will make it easier to build and update new user interfaces in the future.
 
-#### Designers
+** *Note, Catalyst is a work in progress **
```
Comment (rymorgan) [inline styleguide/src/sections/Introduction.md]: Let's remove 'lightweight' here too.

Comment (rymorgan) [inline styleguide/src/sections/Introduction.md]: Remove '*' in front of note.

File: styleguide/src/sections/Introduction.md [hunk 2]
```diff
@@ -1,13 +1,30 @@
-Introducing Reaction Commerce's **Catalyst Design System** and the `@reactioncommerce/catalyst` NPM module, built with React, React Hooks, Material-UI, JSS and the React Testing Library. The **Catalyst Design System** is a lightweight open source design system for building Reaction Admin tools and plugins.
+**Catalyst** is [Reaction Commerce](https://www.reactioncommerce.com/)'s lightweight design system that we use to build Reaction's Admin interface. Catalyst components are used for Reaction Admin and any future operator user interfaces we may build. The Catalyst components are focused on the user experience for operators and will help consistency across the Reaction Admin. Additionally, Catalyst will make it easier to build and update new user interfaces in the future.
```
Comment (machikoyasuda) [inline styleguide/src/sections/Introduction.md]:
```suggestion
**Catalyst** is [Reaction Commerce](https://www.reactioncommerce.com/)'s design system that we use to build Reaction's Admin interface. Catalyst components are used for Reaction Admin and any future operator user interfaces we may build. The Catalyst components are focused on the user experience for operators and will help consistency across the Reaction Admin. Additionally, Catalyst will make it easier to build and update new user interfaces in the future.
```

**Pull request merged**